lib/terminus.js

```diff
@@ -103,6 +110,7 @@ function terminus (server, options = {}) {
     signals = [],
     timeout = 1000,
     healthChecks = {},
+    onSendFailureDuringShutdown = noopResolves,
```

Comment:
do we need a default value for it? if we assign one to it, then this will always be true:

```
if (onSendFailureDuringShutdown) {
    await onSendFailureDuringShutdown()
  }
```

Reply:
Good call.. I'm not sure which is better, to remove the default value or to remove the `if` statement... hmm :thinking: 

I went ahead and removed the default value.

Reply:
Actually I changed my mind, it should look more like the other code that does provide defaults, removing if statement instead lol

Reply:
Whoops, changed my mind again... the if statement is necessary because the `sendFailure` on an actual failed healthcheck does not pass-through the `onSendFailureDuringShutdown`.

Reply:
I think we're good now...